runtime/resolver.go

```diff
@@ -16,7 +22,33 @@ func (r *Resolver) ResolveFunc(module, field string) exec.FunctionImport {
 		case "ext_get_storage_into":
 			return func(vm *exec.VirtualMachine) int64 {
 				log.Debug("executing: ext_get_storage_into")
-				return 0
+				keyData := int(uint32(vm.GetCurrentFrame().Locals[0]))
+				keyLen := int(uint32(vm.GetCurrentFrame().Locals[1]))
+				valueData := int(uint32(vm.GetCurrentFrame().Locals[2]))
+				valueLen := int(uint32(vm.GetCurrentFrame().Locals[3]))
+				valueOffset := int(uint32(vm.GetCurrentFrame().Locals[4]))
+				log.Debug("[ext_get_storage_into]", "local[0]", keyData, "local[1]", keyLen, "local[2]", valueData, "local[3]", valueLen, "local[4]", valueOffset)
```

Comment:
Lets switch these to `keyData` instead of `local[0]` so the logs are more useful